app/Models/User.php

```
@@ -41,7 +51,43 @@ protected function casts(): array
+        return $this->belongsToMany(Task::class, 'task_favorites')
+            ->withTimestamps();
+    }
+
+    public function taskFavorites(): HasMany
+    {
+        return $this->hasMany(TaskFavorite::class);
+    }
+
+    public function isAdmin(): bool
+    {
+        return $this->profile->isAdmin();
```

PHPStan (2.0.3)

Call to an undefined method Illuminate\Database\Eloquent\Model::isAdmin().


(method.notFound)